max_linear_vel: 0.2
min_linear_vel: 0.05
max_angular_vel: 0.35
min_angular_vel: 0.05
gripper_closed_position: 0.8